tasks/loopback_angular.js

```diff
@@ -32,7 +32,8 @@ module.exports = function(grunt) {
     // Merge task-specific and/or target-specific options with these defaults.
     var options = this.options({
       ngModuleName: 'lbServices',
-      apiUrl: undefined
+      apiUrl: undefined,
+      modelsToIgnore: []
```

Comment:
I believe `options.modelsToIgnore` should be handled by the generator, so let's use that as the default value and revert this change.